src/style.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --avenir-family: "Avenir";
+}
+
+@font-face {
+  font-family: "Avenir";
```

Comment:
You created a variable for this parameter, so you should use it wherever possible.

Reply:
if I use it in font face it doesn't work at all